go.mod

```diff
@@ -181,3 +181,9 @@ replace k8s.io/externaljwt => k8s.io/externaljwt v0.32.5
 replace k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.32.5
 
 replace k8s.io/sample-controller => k8s.io/sample-controller v0.32.5
+
+// local pinning
+replace (
+	golang.org/x/crypto => github.com/openshift/golang-crypto v0.33.1-0.20250310193910-9003f682e581
+	golang.org/x/oauth2 => github.com/openshift/golang-oauth2 v0.26.1-0.20250310184649-06a918c6239d
+)
```

Comment:
please check if these are still necessary. The problem was that more recent version of these packages required a too recent golang toolchain to be built. If the golang version we use nowadays is compatible, we can just bump to the fixed version of the u/s package.